tls/data_source_public_key.go

```diff
@@ -0,0 +1,75 @@
+package tls
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourcePublicKey() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourcePublicKeyRead,
+		Schema: map[string]*schema.Schema{
+			"private_key": &schema.Schema{
```

Comment:
Can this be changed to `private_key_pem` to bring it in line with the other resources in this provider?